src/renderer/App.css

```diff
@@ -18,14 +18,18 @@
 
 .banner {
   inline-size: 85%;
-  margin-block: 40px 0;
   margin-inline: auto;
+  margin-block-start: -2px;
```

Comment:
Is `banner` used outside of `ftNotificationBanner`?
If not `ftNotificationBanner` itself got `margin` already
![image](https://github.com/user-attachments/assets/fd3d21ab-24c9-4775-8370-6e4ee0d45c50)

Reply:
That is correct. Nice catch.